akka-actor/src/main/scala-3/akka/util/ByteString.scala

```diff
@@ -0,0 +1,1380 @@
+/*
+ * Copyright (C) 2009-2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.util
+
+import java.io.{ ObjectInputStream, ObjectOutputStream }
+import java.lang.{ Iterable => JIterable }
+import java.nio.{ ByteBuffer, ByteOrder }
+import java.nio.charset.{ Charset, StandardCharsets }
+import java.util.Base64
+
+import scala.annotation.{ tailrec, varargs }
+import scala.collection.{ immutable, mutable }
+import scala.collection.immutable.{ IndexedSeq, IndexedSeqOps, StrictOptimizedSeqOps, VectorBuilder }
+import scala.collection.mutable.{ Builder, WrappedArray }
+import scala.reflect.ClassTag
+
+import scala.annotation.nowarn
+
+object ByteString {
```

Comment:
ByteString is such a compatibility nightmare

Reply:
The entire `scala-3` folder is copy-pasted from `scala-2.13`